SDL/Quintuplet.cu

```diff
@@ -517,13 +549,13 @@ __device__ bool SDL::passChiSquaredConstraint(struct SDL::modules& modulesInGPU,
     }
     else if(layer1 == 1 and layer2 == 2 and layer3 == 3 and layer4 == 4)
     {
-        if(layer5 == 12)
+        if(layer5 == 5)
         {
-            return chiSquared < 0.09461f;
+            return chiSquared < 0.04725f;
         }
-        else if(layer5 == 5)
+        else if(layer5 == 12)
         {
-            return chiSquared < 0.04725f;
+            return chiSquared < 0.09461f;
```

Comment:
Actually this chiSquare is the r-phi one, which is not touched in this PR. I just flip the order because it naturally goes for us to check layer 1,2,3,4,5 first, then check 1,2,3,4,12. Not the vise versa...